tools/please_go/generate/generate.go

```diff
@@ -318,6 +318,9 @@ func (g *Generate) rule(rule *Rule) *bazelbuild.Rule {
 	r := NewRule(rule.kind, rule.name)
 	populateRule(r, rule)
 	r.SetAttr("visibility", NewStringList([]string{"PUBLIC"}))
+	if rule.kind == "go_library" {
```

Comment:
is `cgo_library` a valid kind here too?

Reply:
it doesn't have cover as an arg but maybe it should https://github.com/goddenrich/go-rules/blob/master/build_defs/cgo.build_defs#L3